src/vmm/src/builder.rs

```diff
@@ -233,6 +233,13 @@ fn create_vmm_and_vcpus(
         setup_interrupt_controller(&mut vm)?;
         vcpus = create_vcpus(&vm, vcpu_count, &exit_evt).map_err(Internal)?;
 
+        // Make stdout non blocking.
+        let flags = unsafe { libc::fcntl(libc::STDOUT_FILENO, libc::F_GETFL, 0) };
```

Comment:
GETFL should also be checked against not returning an error, right?

Reply:
I commented [this](https://github.com/firecracker-microvm/firecracker/pull/2461#discussion_r578278679) on the `GETFL` issue. Can we move this discussion there?

Reply:
ok with me